src/util/download.js

```diff
@@ -0,0 +1,58 @@
+import { logWarning } from './logger.js';
+
+const REGEX_URL = /^(http(s?):)([\/|.|\w|\s\:\#\~|-])*/;
```

Comment:
why are you using this complicated url regex? can't you have `/^(http(s?):.*\/\/`?

Reply:
Yes, we I can. The one we currently use worked for validating HTML <img src=...>. as discussed I took away the validation for extension.

Reply:
The regex has been update as you requested!